wherehows-web/app/components/datasets/containers/health-score-gauge.ts

```diff
@@ -0,0 +1,39 @@
+import Component from '@ember/component';
+import { task } from 'ember-concurrency';
+import { IDatasetHealth } from 'wherehows-web/typings/api/datasets/health';
+import { readDatasetHealthByUrn } from 'wherehows-web/utils/api/datasets/health';
+import { get, set } from '@ember/object';
+
+// import { once } from '@ember/runloop';
+
+export default class HealthScoreGauge extends Component {
+  /**
+   * The urn identifier for the dataset
+   * @type {string}
+   */
+  urn: string;
+
+  /**
+   * Defines the tag to be used in the rendered html element for this component
+   * @type {string}
+   */
+  tagName: '';
+
+  /**
+   * The health score to be passed to the container
+   */
+  healthScore: number;
```

Comment:
Maybe initialize this to 0 in the constructor? or type this as `number | void` since its is `undefined`, if only for a second.